playbooks/roles/common_vars/defaults/main.yml

```
@@ -102,6 +102,7 @@ COMMON_MONGO_READ_ONLY_PASS: !!null
 COMMON_ENABLE_DATADOG: False
 # Enable APM monitoring with Datadog (metrics, traces, and logs)
 COMMON_ENABLE_DATADOG_APP: False
+COMMON_ENABLE_SPECIFIC_DATADOG_APPS: []
```

[suggestion] "App" in `COMMON_ENABLE_DATADOG_APP`  to application monitoring as opposed to infrastructure monitoring, so I'd rename to something like this:

```suggestion
COMMON_ENABLE_DATADOG_APP_FOR: []
```

...or something involving "services", maybe.